src/action/common/place_nix_configuration.rs

```diff
@@ -232,6 +236,26 @@ impl PlaceNixConfiguration {
             );
         }
 
+        // NOTE(cole-h): We want to ensure our experimental-features are not clobbered by user
+        // config, so if a user specifies that, we exchange it for the `extra-` variant that just
+        // appends to the list of experimental features.
+        // If the user does indeed want to completely overwrite our experimental features, they can
+        // modify nix.custom.conf to their heart's desire. In most cases, however, they likely just
+        // want their setting to take effect (and probably have no opinion on or would even prefer
+        // our standard experimental features continue to take effect).
```

Comment:
Want to link to the relevant issue here? (https://github.com/DeterminateSystems/nix-installer/issues/1390?)

Reply:
Not really. It doesn't add any additional context that's not already in the comment.

(The only reason I added links for the `trusted-users` one is because that will be deleted once a fix is available in `cachix` and ~widely available, but I want to make sure we double-check all those links before we actually remove that workaround)